payload/src/main.rs

```diff
@@ -28,4 +28,7 @@ fn main() {
     for aux in &mut reader {
         println!("aux: {:?}", aux);
     }
+
+    println!();
+    println!("Good morning, that's a nice Tnetennba!");
```

Comment:
```suggestion
    println!("\nGood morning, that's a nice Tnetennba!");
```

Reply:
Is that considered good practice these days?  Fewer lines, but less legible.  Rejected!

Reply:
`println!()` is preferred.

Technically, `\n` as newline character isn't portable. Though in practice it works pretty much everywhere except Windows.

Reply:
I had no idea! Learned something new.